### ISSUE TITLE
Trashpoint expiration

### ISSUE THREAD

**Issue by krishaamer** [letsdoitworld/World-Cleanup-Day#59]
Change the status of a trashpoint to Outdated on 60 days after updated date

**Comment by lynxlynxlynx** [comment]
Just noting that this will then be the overwhelming status, which perhaps is not desirable.

**Comment by kadry55** [comment]
@rionita Please check what is the period of Trashpoint expiration now. 3months or 6months. September's Trashpoints should be expired if it was 3 months.

**Comment by kadry55** [comment]
![screenshot_20180104-094642](https://user-images.githubusercontent.com/30211382/34555174-1068f3ac-f139-11e7-96de-0eeaffb0281f.png)

Example from september (4m ago) - not expired

**Comment by lynxlynxlynx** [comment]
It doesn't appear to be implemented fully (code keyword: outdated), i.e. nothing is setting this status yet. I guess it would need something cron-like in the db backend to periodically check TP updatedAt against the current date.

Secondly, it's easiest to check if anything has expired by looking through the country stats. There's also an old TP on the south of Ulemiste Jarv lake in Tallinn.

And an idea: scale the expiry time depending on the TP size. So let's say 1mo for category 1 (litter), 2mo for 2, 4mo for 3, 6mo for 4 (truckload or more). Would be more accurate than one value for all.

**Comment by cprpopqual** [comment]
> It doesn't appear to be implemented fully (code keyword: outdated), i.e. nothing is setting this status yet. I guess it would need something cron-like in the db backend to periodically check TP updatedAt against the current date.

This is correct. This feature would need a script to run periodically, check timestamps, and change the status of matching trashpoints. Unfortunately CouchDB does not offer automatic updates like CouchBase, so this needs to be done explicitly from outside the db. It's also not something that should piggyback on other API calls, since it could impact their performance.
